packages/ui-components/src/components/IconTs/Icon.component.tsx

```diff
@@ -830,10 +830,10 @@ type EventHandler = (_event: React.MouseEvent<EventTarget>) => void
 export type IconProps = {
   icon?: KnownIcons
   color?: string
-  size?: number
+  size?: string | number
   title?: string
   className?: string
   href?: string
   disabled?: boolean
   onClick?: EventHandler
-} & (React.HTMLProps<HTMLAnchorElement> | React.HTMLProps<HTMLButtonElement>)
+} & Omit<React.HTMLProps<HTMLAnchorElement> | React.HTMLProps<HTMLButtonElement>, "size">
```

Comment:
Why do you need to omit this type here?

Reply:
size is part of "React.HTMLProps<HTMLAnchorElement> | React.HTMLProps<HTMLButtonElement" and it has type "number", since I need it introduce "number | size" I need to omit the type here.